CONTRIBUTING.md

```diff
@@ -20,7 +20,8 @@ Ever since Moulberry has stopped working on NEU, other contributors have been wo
 
 ### Software configuration
 
-- Clone the NEU repository using `git clone https://github.com/NotEnoughUpdates/NotEnoughUpdates`.
+- Fork the NEU repository using the fork button on top right of the page.
+- Clone the forked repository using `git clone https://github.com/<YourUserName>/<ForkedRepoName>`.
```

Comment:
Maybe specify that they should name the fork NotEnoughUpdates instead of having the placeholder ForkedRepoName?

Reply:
Not that it's a requirement, but it reduces the possibility of some dev thinking he needs to name it something else to work.